doc/tutorial/mutli_arch.hpp

```diff
@@ -0,0 +1,158 @@
+#error This file is for documentation only - DO NOT INCLUDE
+/**
+
+@page multiarch Handling Multiple Architecture Targets
+
+# The Issue
+The **EVE** library uses compile-time code generation process to ensure a high-level of performance.
+However, there can be many reasons why your application or library maybe run on unknown systems.
+Thus, you may want to be able to write your SIMD kernel once but be able to adapt to your end-user's
+hardware.
+
+A first solution could be to just recompile a given kernel with different architecture settings and
+link everything. Then, at runtime, use eve::is_supported to choose the correct implementation
+between the available ones. If the premise of this idea is correct, you'll face issues as the
```

Comment:
Suggest "If the premise of this idea is correct," with "However, in attempting to solve the problem in this way," to avoid confusion about what is meant by "this idea."